matrix/src/dense.rs

```diff
@@ -1056,4 +1056,66 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn test_vertically_packed_row_pair() {
+        use p3_baby_bear::BabyBear;
+        use p3_field::FieldArray;
+
+        type Packed = FieldArray<BabyBear, 2>;
+
+        let matrix = RowMajorMatrix::new(
+            vec![
+                BabyBear::new(1),
+                BabyBear::new(2),
+                BabyBear::new(3),
+                BabyBear::new(4), // Row 0
+                BabyBear::new(5),
+                BabyBear::new(6),
+                BabyBear::new(7),
+                BabyBear::new(8), // Row 1
+                BabyBear::new(9),
+                BabyBear::new(10),
+                BabyBear::new(11),
+                BabyBear::new(12), // Row 2
+                BabyBear::new(13),
+                BabyBear::new(14),
+                BabyBear::new(15),
+                BabyBear::new(16), // Row 3
+            ],
```

Comment:
Given you have the matrix visualization below, you can simplify this vector declaration. Something like:
```suggestion
            (1..17).map(BabyBear::from_int).collect::<Vec<_>>(),
```
should work,

Reply:
fixed